model_compression_toolkit/core/common/mixed_precision/mixed_precision_search_manager.py

```diff
@@ -121,7 +125,10 @@ def compute_min_ru(self) -> Dict[RUTarget, np.ndarray]:
         for ru_target, ru_fns in self.compute_ru_functions.items():
             # ru_fns is a pair of resource utilization computation method and 
             # resource utilization aggregation method (in this method we only need the first one)
-            min_ru[ru_target] = ru_fns[0](self.min_ru_config, self.graph, self.fw_info, self.fw_impl)
+            if ru_target is RUTarget.ACTIVATION:
```

Comment:
is "is" correct? usually enum is compared with '==' (I think it is equivalent for enums but just want to make sure)

Reply:
it's the same. "is" feels clearer